packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -70,9 +70,10 @@
 	margin-left: auto;
 	margin-right: auto;
 
-	// Space title similarly to other blocks.
-	// This uses negative margins so as to not affect the default block margins.
-	margin-bottom: -$block-padding - $block-spacing - $border-width - $border-width;
+	// Apply default block margin below the post title.
+	// This ensures the first block on the page is almost in a good position.
+	// This rule can be retired once the title becomes an actual block.
+	margin-bottom: $default-block-margin;
```

Comment:
This has the side-effect of always providing this much space down to the first block:

![Screenshot 2019-03-13 at 13 27 59](https://user-images.githubusercontent.com/1204802/54278845-dc4c2e80-4593-11e9-9531-00f91f973cb5.png)

Compare with master:

![Screenshot 2019-03-13 at 13 28 35](https://user-images.githubusercontent.com/1204802/54278864-ecfca480-4593-11e9-87be-fb8e67df6d34.png)

But this is actually the simplest and cleanest way to provide a consistent experience without resorting to crazy first-child and conditional rules. Consider a situation where the first block inserted is not born with a top-margin. Or consider one where the first block inserted has a HUGE intentional top margin. If we zero that margin out to override and provide a different one, we cancel that out.

By providing this slightly larger margin, we avoid all that happening. If we are passionate about keeping the current behavior while improving things, the real way to go here is to convert the title to an actual block, and not just a "faux" block as it is right now.

Reply:
This rule seems ok, especially when we consider that _if_ the first Block does have a margin and it is _larger_ than the bottom margin on the page title then margin collapsing should ensure that it doesn't double up.

Block shouldn't have huge top margins, but if they do then perhaps the Block author should be responsible for ensuring they play nicely in these types of situations?

Reply:
This seems 100% fine to me, for all the reasons @getdave mentioned. And frankly, I like the appearance of the little bit of extra margin up there. 😄

Reply:
I say remove all styles that have padding and margins for blocks. Only style the UI elements of the editor.